pkg/kubelet/kuberuntime/labels.go

```diff
@@ -106,15 +96,6 @@ func newPodAnnotations(pod *v1.Pod) map[string]string {
 		annotations[k] = v
```

Comment:
This loop was added to handle this special case, should we put it back to what it was before? https://github.com/kubernetes/kubernetes/commit/fd94032b2132e740a70449ffb3e9fc8f66d2c9d4#diff-a7e9fc1284dd3aa38fe530c010397a14d657f4b81e01217344c053ad6d5454a3L92

Reply:
Yea, I can do that. Good catch!